package.json

```diff
@@ -11,18 +11,25 @@
   "main": "./index.js",
   "dependencies": {
     "any-shell-escape": "^0.1.1",
+    "bufferstreams": "^1.1.1",
     "gulp-util": "^3.0.6",
-    "require-dir": "^0.1.0",
-    "through2": "^0.6.5"
+    "require-dir": "^0.3.1",
+    "strip-bom-stream": "^3.0.0",
+    "through2": "^2.0.3",
+    "vinyl": "^2.0.1"
   },
   "devDependencies": {
-    "jshint": "^2.8.0",
-    "mocha": "^2.3.0",
+    "eslint": "^3.15.0",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^3.0.1",
+    "husky": "^0.13.1",
+    "mocha": "^3.2.0",
     "rimraf": "^2.4.3",
-    "should": "^7.1.0"
+    "should": "^11.2.0"
   },
   "scripts": {
-    "test": "mocha --reporter spec --timeout 6000 test/main.js && jshint ./index.js ./examples/. ./lib/. ./test/."
+    "test": "mocha --reporter spec --timeout 6000 test/main.js && gulp precommit",
+    "precommit": "gulp precommit"
```

Comment:
Switch `gulp precommit` to `npm run lint` and add a new script like `test` called lint which will run eslint